pkg/aws/testutil_test.go

```diff
@@ -186,6 +222,43 @@ func fakeRDSClient(modifyFn func(c *rdsClientMock) error) (*rdsClientMock, error
 	return client, nil
 }
 
+type mockEc2Client struct {
```

Comment:
We probably should look to move this struct and the mocks to this directory https://github.com/integr8ly/cluster-service/blob/master/pkg/aws/testutil_test.go#L1 as it seems to store the mocks for RDS, Elasticache and S3